scripts/build/Platform/Windows/build_config.json

```diff
@@ -36,6 +36,28 @@
       "test_cpu_profile_vs2019"
     ]
   },
+  "periodic_test_profile_vs2019_pipe": {
```

Comment:
this name is too generic, should we name it with more information? as it is for aws automation test?

Reply:
Shouldn't the vs version be a parameter that comes in? If we get VS 2021 (which is currently in preview) would we duplicate this setup?

Reply:
This is not for aws automation test only. Define this pipeline to make sure that aws resources are deployed before we run all the periodic tests. I removed the "nightly-incremental" and "nightly-clean" tags on the "periodic_test_profile_vs2019" job to avoid it running on the nightly build since it needs to be replaced by the "periodic_test_profile_vs2019_pipe" job.

Reply:
Seems like the current build configuration are all hardcoding the vs version. You can copy this block and change the corresponding vs version to define a new job. I agree that we should consider to make vs version a parameter for all the Jenkins job here.

Reply:
Removed the pipeline and add a new tag so that the resource deployment can be run on a separate Jenkins job before nightly build.